docs/resources/claims_mapping_policy.md

```diff
@@ -70,8 +70,9 @@ The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/l
 
 ## Import
 
-Claims Mapping Policy can be imported using the `id`, e.g.
+Claims Mapping Policy can be imported using the object ID of the associated service principal and the ID of the claims mapping policy, e.g.
 
 ```shell
-terraform import azuread_claims_mapping_policy.my_policy 00000000-0000-0000-0000-000000000000
+terraform import azuread_claims_mapping_policy.my_policy /servicePrincipals/00000000-abcd-0000-0000-000000000000/claimsMappingPolicies/00000000-efgh-0000-0000-000000000000
```

Comment:
I think this should be:
```suggestion
terraform import azuread_claims_mapping_policy.my_policy /policies/claimsMappingPolicies/00000000-0000-0000-0000-000000000000
```
This is a different resource/ID  to `azuread_service_principal_claims_mapping_policy_assignment`